src/components/Ledgercard/index.css

```diff
@@ -0,0 +1,226 @@
+html{
+    overflow: hidden;
+}
+
+.ledger-card{
+    display: grid;
+    cursor: pointer;
+}
+
+.editor{
+    height: 20rem;
+      
+}
+.color-editor-icon{
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.img-res{
+    width: 100%;
+    height: max-content;
+}
+
+.ledger-card:hover .img-res{
+    opacity: 0.1;
+    transition: opacity 0.6s;
+}
+
+.ledger-card:hover .ledger-card-desc{
+    opacity: 1;
+    transition: opacity 0.5s;
+    border-style: solid;
+    border-width: 1px;
+    padding-left: 5px;
+}
+
+.grid-overlap{
+    grid-column: 1/-1;
+    grid-row: 1/-1;
+}
+
+.ledger-card-desc{
+    opacity: 0;
+    transition: opacity 1s;
+}
+
+/* button should go into one component */
+
+.ledger-note{
+    text-align: left;
+    color:var(--pale-white);
+    padding: 1.5em ;
+    display: grid;  
+}
+
+.color-picker {
+    font-size:20px;
+    height:24px;
+    width: 24px;
+    padding: 0px;
+    margin-right: 5px;
+    border-style: none;
+    background-color: transparent;
+}
+
+.ledger-note-actions{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    margin-left:auto;
+}
+
+.ledger-note-actions-wrapper{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.note-proiority-label{
+    background-color: #FF004D;
+    width: max-content;
+    height: max-content;
+    padding:5px;color: #DBD8E3;;
+    grid-area: 1/-1;
+    opacity: 0.8;
+    margin-top:-15px;
+    margin-right:10px;
+    margin:-15px 10px 10px 0px;
+}
+
```

Comment:
Why using margin-top and margin-right? You're anyways using the margin shorthand proeprty